app/src/main/res/values/settings_keys.xml

```diff
@@ -158,7 +158,7 @@
     <string name="caption_user_set_key" translatable="false">caption_user_set_key</string>
 
     <!-- Content & History -->
-    <string name="show_search_suggestions_key" translatable="false">show_search_suggestions</string>
+    <string name="show_remote_search_suggestions_key" translatable="false">show_search_suggestions</string>
```

Comment:
Why change this? For all of them, it's whatever_key without changes in whatever. We couldn't change the whatever either, as then a previously set setting would be back to its default.

Reply:
Yeah, I only changed the string name to make it more clear that the option only refers to remote suggestions, but I left the value as it was before as not to reset settings